package.json

```diff
@@ -67,6 +73,12 @@
       }
     ],
     "menus": {
+      "explorer/context": [
+        {
+          "command": "onevscode.show-compile-webview",
+          "when": "resourceExtname in onevscode.compilableExtList || resourcePath in onevscode.compilableDirList"
```

Comment:
ref: https://code.visualstudio.com/api/references/when-clause-contexts#in-conditional-operator